dvd/dvd_dump.ixx

```diff
@@ -451,6 +465,119 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
             for(uint32_t i = 0; i < manufacturer_structures.size(); ++i)
                 write_vector(std::format("{}{}.manufacturer", image_prefix, manufacturer_structures.size() > 1 ? std::format(".{}", i) : ""), manufacturer_structures[i]);
 
+            if(is_xbox)
+            {
+                std::vector<uint8_t> security_sector(0x800);
+
+                // retry get security sector
+                for(uint32_t i = 0;; i++)
+                {
+                    status = cmd_kreon_get_security_sector(*ctx.sptd, security_sector);
+                    if(!status.status_code)
+                        break;
+                    else if(i == 10)
+                        throw_line("failed to get security sectors, SCSI ({})", SPTD::StatusMessage(status));
+                    LOG("failed to get security sector, retrying {}/10", i + 1);
```

Comment:
I think this would be 11 times (0-10) and you would want `i == 9` to only check 10 times. Same for the refine case

Reply:
Yes you're right, it's an initial pass and then 10 retries. Although 10 may be too large anyway.